packages/dev/gui/src/2D/controls/rectangle.ts

```diff
@@ -3,11 +3,13 @@ import type { Measure } from "../measure";
 import { RegisterClass } from "core/Misc/typeStore";
 import { serialize } from "core/Misc/decorators";
 import type { ICanvasRenderingContext } from "core/Engines/ICanvas";
+import { Vector4 } from "core/Maths/math";
 
 /** Class used to create rectangle container */
 export class Rectangle extends Container {
     private _thickness = 1;
     private _cornerRadius = 0;
+    private _cornerRadiusFree = Vector4.Zero();

```

Comment:
apart from setAll, is there a reason you are not using an array of numbers for this?

Reply:
i think a vector is redundant. there is no meaning to x,y,z or w. it will be simpler and more readable to use a simple array